mesonbuild/interpreter/type_checking.py

```diff
@@ -577,6 +597,17 @@ def _name_validator(arg: T.Optional[T.Union[str, T.List]]) -> T.Optional[str]:
         validator=in_set_validator(set(compilers.all_languages)),
         since='0.51.0',
     ),
+    KwargInfo(
+        'objects',
+        ContainerTypeInfo(list, (str, File, CustomTarget, CustomTargetIndex, GeneratedList)),
+        listify=True,
+        default=[],
+        validator=_objects_validator,
+        since_values={
+            ContainerTypeInfo(list, (GeneratedList, CustomTarget, CustomTargetIndex)):
+                ('1.1.0', 'generated sources as positional "objects" arguments')
```

Comment:
Apparently we never had a FeatureNew for this. :(

Reply:
No we didn't :/